packages/ui-range-input/src/RangeInput/index.tsx

```diff
@@ -212,16 +209,12 @@ class RangeInput extends Component<RangeInputProps, RangeInputState> {
               this._input = c
             }}
             type="range"
-            role="slider"
             id={this.id}
             min={this.props.min}
             max={this.props.max}
             step={this.props.step}
             value={this.value}
             onChange={this.noopChange}
-            aria-valuenow={this.value}
-            aria-valuemin={this.props.min}
-            aria-valuemax={this.props.max}
```

Comment:
Again, native HTML is recommended:  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role : "It is recommended to use a native [input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) of type range, [input type="range"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range), rather than the slider role."